Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -226,7 +226,10 @@ extension ChannelDTO {
 extension _ChatChannel {
     /// Create a ChannelModel struct from its DTO
     fileprivate static func create(fromDTO dto: ChannelDTO) -> _ChatChannel {
-        let members: [_ChatChannelMember<ExtraData.User>] = dto.members.map { $0.asModel() }
+        let memberCompare: (MemberDTO, MemberDTO) -> Bool = { lhs, rhs in
+            lhs.user.lastActivityAt ?? Date(timeIntervalSince1970: 0) < rhs.user.lastActivityAt ?? Date(timeIntervalSince1970: 0)
+        }
+        let members: [_ChatChannelMember<ExtraData.User>] = dto.members.sorted(by: memberCompare).prefix(25).map { $0.asModel() }
```

Comment:
The reason why we didn't have this parametrized was that we could find a simple way how to pass the value from the config here. But then it came to me and of course, Apple didn't let us down! https://developer.apple.com/documentation/coredata/nsmanagedobjectcontext/1506740-userinfo

This will probably have to set in `DatabaseContainer` based on the values provided by `ChatCient`.